x-pack/filebeat/module/suricata/eve/_meta/fields.yml

```diff
@@ -9,8 +9,51 @@
       type: ip
     - name: flow_id
       type: keyword
-    # Geo Points should be the ECS fields, but no way to config from here.
-    - name: src_geo.location
-      type: geo_point
-    - name: dest_geo.location
-      type: geo_point
+
+    - name: src_geo
+      type: group
+      fields:
+        - name: continent_name
+          type: keyword
+          description: >
+            Name of the continent.
+        - name: country_iso_code
+          type: keyword
+          description: >
+            Country ISO code.
+        - name: location
+          type: geo_point
+          description: >
+            Longitude and latitude.
+        - name: region_name
```

Comment:
By default for 6.5 it looks like ingest node geoip processor will also add `region_iso_code`. https://www.elastic.co/guide/en/elasticsearch/plugins/6.x/using-ingest-geoip.html